doc/classes/RayCast2D.xml

```diff
@@ -36,7 +36,7 @@
 		<method name="force_raycast_update">
 			<return type="void" />
 			<description>
-				Updates the collision information for the ray. Use this method to update the collision information immediately instead of waiting for the next [code]_physics_process[/code] call, for example if the ray or its parent has changed state.
+				Updates the collision information for the ray immediately, without waiting for the next [code]_physics_process[/code] call. The method may be helpful, for example, when the ray or its parent has changed state.
```

Comment:
The original wording of "Use this method" should be retained, the hypothetical "may be useful" isn't correct for instructions IMO

Reply:
Fixed.